pom.xml

```diff
@@ -152,7 +152,7 @@
       <!-- NB: We want this, despite warning from dependency:analyze. -->
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
-      <version>1.2.0</version>
+      <version>1.2.9</version>
```

Comment:
Since `logback-core` is a transitive dependency of `logback-classic`, would it make sense to combine the two dependencies in the POM?
This would have the advantage of reducing the chances of having these getting out of sync and the resolution issues associated with different build systems